src/BulletGauge/BulletGauge.tsx

```diff
@@ -0,0 +1,73 @@
+import { BulletGaugeProps } from "./BulletGauge.types";
+import Plotly from "react-plotly.js";
+import React from "react";
+import { useTheme } from "@mui/material";
+
+const BulletGauge = ({ title, value, suffix }: BulletGaugeProps) => {
+  // theme hook
+  const theme = useTheme();
+
+  return (
+    <Plotly
+      data={[
+        {
+          domain: { x: [0, 1], y: [0, 1] },
+          gauge: {
+            axis: {
+              range: [null, 100],
+              tickfont: {
+                color: theme.palette.mode === "light" ? "black" : "white",
+                size: 12
+              }
+            },
+            bar: {
+              color: value < 30 ? "#f44336" : value > 70 ? "#4caf50" : "#ff9800"
```

Comment:
We should probably avoid hardcoding colours into components. Do we have colours in our palette that would work, if not could these be added? Traffic light colours would likely have re-use in lots of places.

Reply:
We could make use of the `theme.palette` colours like this

```
            bar: {
              color:
                value < 30
                  ? theme.palette.error.main
                  : value > 70
                    ? theme.palette.success.main
                    : theme.palette.warning.main
            },
```

Reply:
Yep I think that would be better